src/frontend/helpers/virtualKeyboard.ts

```diff
@@ -2,47 +2,61 @@ import Keyboard from 'simple-keyboard'
 import 'simple-keyboard/build/css/index.css'
 
 let virtualKeyboard: Keyboard | null = null
+let targetInput: HTMLInputElement | null = null
 
 function currentElement() {
   return document.querySelector<HTMLElement>(':focus')
 }
 
-function searchInput() {
-  // only change this if you change the id of the input element
-  // in frontend/components/UI/SearchBar/index.tsx
-  return document.querySelector<HTMLInputElement>('#search')
-}
-
 function focusKeyboard() {
   const firstButton = document.querySelector<HTMLElement>(
     '.hg-button[data-skbtn="h"]'
   )
   firstButton?.focus()
 }
 
-function typeInSearchInput(button: string) {
-  const input = searchInput()
-  if (!input) return
+function typeInInput(button: string) {
+  if (!targetInput) return
 
   if (button.length === 1) {
-    input.value = input.value + button
+    targetInput.value = targetInput.value + button
   } else if (button === '{bksp}') {
-    if (input.value.length > 0) {
-      input.value = input.value.slice(0, -1)
+    if (targetInput.value.length > 0) {
+      targetInput.value = targetInput.value.slice(0, -1)
     }
   } else if (button === '{space}') {
-    input.value = input.value + ' '
+    targetInput.value = targetInput.value + ' '
   }
-  input.dispatchEvent(new Event('input'))
+  targetInput.dispatchEvent(new Event('input'))
+}
+
+function makeKeyboardTopLayer() {
+  const wrapper = document.querySelector(
+    '.simple-keyboard-wrapper'
+  ) as HTMLDialogElement
```

Comment:
```suggestion
  const wrapper = document.querySelector<HTMLDialogElement>(
    '.simple-keyboard-wrapper'
  )
```
This probably won't lint

Reply:
updated